index.html

```diff
@@ -353,6 +354,69 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+            <div id="fifthTab" class="tab-pane fade"  >
+            <div class="row cdisplayFlex" style="background-color: #f5f5f5;">
+                <div class="col-lg-12 col-md-12 col-sm-12 focusPane" style="padding:20px;border: 1px #c7c7c7 solid;">
+                    <div class="FullCode" style="background-color: #f5f5f5;">
+                        <div class="highlight">
+                            <pre class="language-ballerina basic ballerina">
+                                <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerinax/kafka;
+import ballerina/lang.value;
+
+configurable string groupId = "group-id";
+configurable string listeningTopic = "topic-orders";
+configurable string publishingTopic = "topic-success-orders";
+configurable decimal pollingInterval = 1;
+configurable string kafkaEndpoint = kafka:DEFAULT_URL;
+
+type Order readonly & record {|
```

Comment:
Can we improve the field names to match the real-world purchase order fields? 

type Order readonly & record {|
    int id;
    string desc;
    PaymentStatus paymentStatus;
|};

Reply:
Added in https://github.com/ballerina-platform/ballerina-dev-website/pull/3735/commits/76f775c43a7430c59f572264b8444345bf1687c8